.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# What does this change intend to accomplish?
+
+ 
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### New Feature Submissions:
+
+* [ ] Does your submission pass tests?
+* [ ] Have you lint your code locally prior to submission?
+
+### Changes to Core Features:
+
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you written new tests for your core changes, as applicable?
+* [ ] Have you successfully ran tests with your changes locally?
+
```

Comment:
Have you updated the documentation/user manual, as applicable?

Reply:
Oooo I like this one. Also, have you updated the changelog, as applicable.

Reply:
implemented~